package.json

```diff
@@ -1,14 +1,30 @@
 {
-  "name": "activitypods-shapes",
+  "name": "activitypods-shapes-monorepo",
+  "private": true,
+  "type": "module",
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "start": "node index.js"
+    "start": "node index.ts",
+    "build": "lerna run build",
+    "prepublishOnly": "yarn build"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "dotenv": "~16.4.7",
+    "dotenv": "^16.5.0",
     "express": "^4.18.2",
     "negotiator": "^1.0.0",
     "rdf-transform": "^1.0.0",
     "streamify-string": "^1.0.1"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.21",
+    "@types/negotiator": "^0.6.3",
+    "@types/streamify-string": "^1.0.4",
+    "lerna": "^8.2.2",
+    "parcel": "^2.15.0",
+    "typescript": "^5.8.3",
+    "yalc": "^1.0.0-pre.53"
   }
```

Comment:
Since node v24 is required, we should indicate this here (it does not work at all with v22)